src/goalServices/changeGoalStatus.test.js

```diff
@@ -54,12 +54,13 @@ describe('changeGoalStatus service', () => {
   });
 
   afterAll(async () => {
-    await db.User.destroy({ where: { id: mockUser.id } });
-    await db.Goal.destroy({ where: { id: goal.id } });
+    await db.Goal.destroy({ where: { id: goal.id }, force: true });
+    await db.GrantNumberLink.destroy({ where: { grantId: grant.id }, force: true });
```

Comment:
`force: true` because we don't just want to set `deletedAt`, we want to actually remove these rows.

`GrantNumberLink` removed before `Grant` because the grant fails to remove due to a constraint.